packages/office-ui-fabric-react/src/components/Coachmark/PositioningContainer/PositioningContainer.styles.ts

```diff
@@ -1,5 +1,6 @@
 import { memoizeFunction } from '../../../Utilities';
-import { mergeStyleSets, focusClear, IStyle, HighContrastSelector } from '../../../Styling';
+import { mergeStyleSets, focusClear, HighContrastSelector } from '../../../Styling';
+import type { IStyle } from '../../../Styling';
```

Comment:
This one needs to be reverted--v7 doesn't support type imports
```suggestion
import { mergeStyleSets, focusClear, IStyle, HighContrastSelector } from '../../../Styling';
```